frontend/app/src/features/room/components/info/DifficultyBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import { DifficultyType } from "@/constants/question"
+import { Badge } from "@chakra-ui/react"
+
+interface Props {
+    difficulty: DifficultyType
+}
+
+
+
+export const DifficultyBadge: React.FC<Props> = ({ difficulty }) => {
+
+    const colorSchemes: Record<DifficultyType, string> = {
+        "Easy": "green",
+        "Medium": "yellow",
+        "Hard": "red"
+    }
+
+    return <Badge colorScheme={colorSchemes[difficulty]} variant="outline">Easy</Badge>
```

Comment:
```suggestion
    return <Badge colorScheme={colorSchemes[difficulty]} variant="outline">{difficulty}</Badge>
```

did u mean to do this?

Reply:
yup LOL i forgot to change that part